controllers/global_search_setup.go

```diff
@@ -339,13 +341,25 @@ func (r *SearchReconciler) createManifestWork(ctx context.Context, cluster strin
 								},
 							},
 						},
+						map[string]interface{}{
+							"apiVersion": "v1",
+							"kind":       "ConfigMap",
+							"metadata": map[string]interface{}{
+								"name":      "search-hub-name",
```

Comment:
Can we use a more general name like `search-global-config`?  This makes easier to reuse if we need to add more data in the future.